frontend/src/types/types.ts

```diff
@@ -0,0 +1,47 @@
+import { FileResult } from './zod/authorship-type';
+import {
+  AuthorFileTypeContributions,
+  CommitResultRaw,
+  Commits,
+} from './zod/commits-type';
+import { RepoRaw } from './zod/summary-type';
+
+// We add these three fields in setContributionOfCommitResultsAndInsertRepoId of utils/api.ts
+export interface CommitResult extends CommitResultRaw {
+  repoId: string;
+  insertions: number;
+  deletions: number;
+}
+
+// Similar to AuthorDailyContributions, but uses the updated CommitResult with the three new fields
+export interface DailyCommit {
+  commitResults: CommitResult[];
+  date: String;
```

Comment:
Thanks for catching this! I realised that our current eslint configuration doesn't enforce any typescript rules for `.ts` files. I'll make a separate PR for this, since it'll require a decent amount of refactoring to remove the `any` types that eslint complains about, and I added some `any` definitions in this PR to minimize changes to the current code.